backend/frontend/templates/frontend/auth/page_permission_denied.html

```diff
@@ -0,0 +1,13 @@
+{% extends "frontend/base_logged_in.html" %}
+{% block content %}
+  <div class="mx-auto w-1/3 p-6 bg-white rounded my-4">
+    <h1 class="{{ styles.heading1 }} text-center">Permission Denied</h1>
```

Comment:
For consistency's sake, let's align the heading to the left like similar auth templates.